src/main/java/frc/robot/Constants.java

```diff
@@ -185,8 +206,8 @@
     public static final double CLIMB_TO_NEXT_BAR_TIMEOUT = 12.0;
 
     // robot starting distance
-    public static final double STARTING_DISTANCE = 74.0;//TARMAC_DEFAULT_DISTANCE - 10.0;
-
+    public static final double STARTING_DISTANCE = TARMAC_DEFAULT_DISTANCE - 10.0;
```

Comment:
Was hand adjusted at Revere. Please leave as was.

Reply:
Complete with the misleading comment?